dapps/profile.vue/src/i18n/de.json

```diff
@@ -148,6 +149,21 @@
         }
       }
     },
+    "did": {
+      "created-at": "erstellt am",
+      "choose-contact": "Kontakt auswählen",
+      "delegates-desc": "Im Folgenden können Delegates ausgewählt werden, die im Namen der Identität agieren dürfen. Diese sind beispielsweise befugt Informationen editieren, Daten freigeben oder Transaktionen durchführen.",
```

Comment:
Do German speaking understand "delegates" or should we call it something else? On the other side, the ui will be probably used by more technically experienced users, so it would be okay.

Reply:
I talked to Daniel and we will keep the technical stuff (DID, delegates, controllers) in English for now.